rover/src/owr_messages/msg/science.msg

```diff
@@ -0,0 +1,9 @@
+uint32 temperature
```

Comment:
It might be a good idea to add a ros Header as well

Reply:
Done

Reply:
It was causing compile errors in embedded, so I decided to leave that out in the interest of debugging time